src/Controller/BlogController.php

```diff
@@ -92,7 +93,7 @@ public function postShow(Post $post): Response
      * (postSlug) doesn't match any of the Doctrine entity properties (slug).
      * See https://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/converters.html#doctrine-converter
      */
-    public function commentNew(Request $request, Post $post, EventDispatcherInterface $eventDispatcher): Response
+    public function commentNew(Request $request, Post $post, EventDispatcherInterface $eventDispatcher, ManagerRegistry $doctrine): Response
```

Comment:
You could even inject the `EntityManagerInterface` directly.

Reply:
Much better. Done!